pom.xml

```diff
@@ -1,169 +1,225 @@
+		<developerConnection>git@github.com:roskenet/springboot-javafx-support.git</developerConnection>
+		<url>https://github.com/roskenet/springboot-javafx-support</url>
+	</scm>
 
-  <developers>
-    <developer>
-      <name>Felix Roske</name>
-      <email>f.roske@roskenet.de</email>
-      <url>http://www.felixroske.de</url>
-      <organization>roskenet</organization>
-    </developer>
-  </developers>
+	<developers>
+		<developer>
+			<name>Felix Roske</name>
+			<email>f.roske@roskenet.de</email>
+			<url>http://www.felixroske.de</url>
+			<organization>roskenet</organization>
+		</developer>
+	</developers>
 
-  <licenses>
-    <license>
-      <name>MIT</name>
-    </license>
-  </licenses>
+	<licenses>
+		<license>
+			<name>MIT</name>
+		</license>
+	</licenses>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-context</artifactId>
-      <version>${spring-framework.version}</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot</artifactId>
-      <version>${spring-boot.version}</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-      <version>${slf4j.version}</version> 
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>de.roskenet</groupId>
-      <artifactId>springboot-javafx-test</artifactId>
-      <version>1.0.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-test</artifactId>
-      <version>${spring-framework.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.12</version>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context</artifactId>
+			<version>${spring-framework.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot</artifactId>
+			<version>${spring-boot.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<version>${spring-boot.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+			<version>${slf4j.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-test</artifactId>
+			<version>${spring-framework.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
```

Comment:
As far as I understand, you want to include test helper classes into this jar, so that users only need to define one dependency?
This will not work like this. Classes under src/test won't be assembled into the release jar.
You could add GUITest helper classes to src/main but this would add them into every application using the springjfx jar, which is something you shouldn't do.
Keep the test helper seperately.